### PR TITLE
convert splitting and sliding windows

### PR DESCRIPTION
**Pull request by davetroiano**
Also make tutorial running sections consistent

### CODE REVIEW

File: splitting/ksql/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# How to split a stream of events into substreams with ksqlDB
+
+
+If you have a stream of events in a Kafka topic and wish to split it into substreams based on a field in each event (a.k.a. content-based routing), you an use ksqlDB's [WHERE](https://docs.ksqldb.io/en/latest/developer-guide/ksqldb-reference/quick-reference/#where) clause to accomplish the task.
+
+For example, suppose that you have a Kafka topic representing appearances of an actor or actress in a film, with each event also containing the movie genre. The following CSAS (`CREATE STREAM AS SELECT`) queries will split the stream into three substreams: one containing drama events, one containing fantasy, and one containing events for all other genres:
+
```
Comment (bbejeck) [inline splitting/ksql/README.md:7]: I'd show the `ACTINGEVENTS` stream to eliminate any confusion on where the data is coming from.

**Review by bbejeck — DISMISSED** [review]
Nice work @davetroiano, just one comment otherwise this LGTM

File: splitting/ksql/README.md (hunk 1 above)
Comment (davetroiano) [reply splitting/ksql/README.md:14]: added this

**Review by bbejeck — APPROVED** [review]
LGTM

**Pull request merged**